### ISSUE TITLE
Enhance AppLaud project creation wizard to allow selection from multiple bundled PhoneGap versions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#74]
```
More critical if there are more poor quality releases like Cordova 1.5.0


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 19 Mar 2012 at 10:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 14 Apr 2012 at 4:08
- Added labels: **Milestone-Release1.2.9**

**Comment by GoogleCodeExporter** [comment]
```
Addressed in 1.2.9
```

Original comment by `paul.beu...@gmail.com` on 19 Apr 2012 at 12:56
- Changed state: **Fixed**
